package.json

```diff
@@ -103,7 +103,7 @@
     "regl": "^1.3.7",
     "regl-error2d": "^2.0.5",
     "regl-line2d": "^3.0.12",
-    "regl-scatter2d": "^3.0.6",
+    "regl-scatter2d": "git://github.com/a-vis/regl-scatter2d.git#0515a47c5468363d19654fcb9383d7135c93c8ad",
```

Comment:
https://github.com/gl-vis/regl-scatter2d/compare/plotly-issue-3232

Reply:
So this brings the max number of colors in a WebGL texture to 256x256=65536, and falls back to plain buffer beyond that? That's amazing! I gotta be honest, I don't fully understand your `split8Uint` dance, maybe you could explain it to me in person before or after the meeting tomorrow?

Now, looks like your latest commit made several image test cases fail, do you know why that is?

Reply:
Yes I would fix this today. Don't worry about the failed image test for the moment. There are a bunch of commits which are not yet pushed to gl-vis.